backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadService.java

```diff
@@ -1,75 +1,41 @@
 package gov.cdc.usds.simplereport.service;
 
-import static gov.cdc.usds.simplereport.api.Translators.parsePersonRole;
-import static gov.cdc.usds.simplereport.api.Translators.parsePhoneType;
-import static gov.cdc.usds.simplereport.api.Translators.parseUserShortDate;
-import static gov.cdc.usds.simplereport.api.Translators.parseYesNo;
-import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.convertEthnicityToDatabaseValue;
-import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.convertRaceToDatabaseValue;
-import static gov.cdc.usds.simplereport.validators.CsvValidatorUtils.convertSexToDatabaseValue;
-
-import com.fasterxml.jackson.databind.MappingIterator;
 import gov.cdc.usds.simplereport.api.model.errors.CsvProcessingException;
 import gov.cdc.usds.simplereport.api.model.filerow.PatientUploadRow;
 import gov.cdc.usds.simplereport.api.uploads.PatientBulkUploadResponse;
 import gov.cdc.usds.simplereport.config.AuthorizationConfiguration;
-import gov.cdc.usds.simplereport.db.model.Facility;
-import gov.cdc.usds.simplereport.db.model.Organization;
-import gov.cdc.usds.simplereport.db.model.Person;
-import gov.cdc.usds.simplereport.db.model.PhoneNumber;
-import gov.cdc.usds.simplereport.db.model.auxiliary.StreetAddress;
 import gov.cdc.usds.simplereport.db.model.auxiliary.UploadStatus;
 import gov.cdc.usds.simplereport.service.model.reportstream.FeedbackMessage;
-import gov.cdc.usds.simplereport.validators.CsvValidatorUtils;
 import gov.cdc.usds.simplereport.validators.FileValidator;
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
 import java.util.UUID;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
-import org.springframework.transaction.annotation.Transactional;
 
 /**
  * Service to upload a roster of patient data given a CSV input. Formerly restricted to superusers
- * but now (almost) available to end users.
+ * but now available to end users.
  *
- * <p>Updated by emmastephenson on 10/24/2022
+ * <p>Updated by emmastephenson on 12/2/2022
  */
 @Service
-@Transactional
 @RequiredArgsConstructor
 @Slf4j
 public class PatientBulkUploadService {
 
-  private final PersonService _personService;
-  private final AddressValidationService _addressValidationService;
-  private final OrganizationService _organizationService;
   private final FileValidator<PatientUploadRow> _patientBulkUploadFileValidator;
+  private final PatientBulkUploadServiceAsync _patientBulkUploadServiceAsync;
 
-  // This authorization will change once we open the feature to end users
-  @AuthorizationConfiguration.RequireGlobalAdminUser
+  @AuthorizationConfiguration.RequirePermissionCreatePatientAtFacility
```

Comment:
is this authorization change premature here? my understanding is the feature will be available to org or facility managers, which this change would turn on, right? Do we want to wait to update this or go ahead?

Reply:
Good catch - it is a little premature, and this will indeed open up the endpoint to org/facility managers. The reason I made the change is because having the existing permissions made testing nearly impossible - you needed a user that had global permissions to add the patients, but then to verify the patients were added you needed a user with permission to view the patients at a facility. Rather than try to work that permissioning out I just made this change, since it was next up in the work stream regardless. There's still no UI available for it (that little checkbox on the frontend does check for superuser permissions before displaying), so org/facility users shouldn't notice a change. They would be able to hit the endpoint directly with a CURL request, but I still think that's a low-risk enough activity to allow for now.

Reply:
Yeah I thought I remembered this was necessary for tests. I'm fine with it, we may just be able to close the other ticket out at the same time.